src/measure_theory/integral/bochner.lean

```diff
@@ -3,8 +3,7 @@ Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Zhouhang Zhou, Yury Kudryashov, Sébastien Gouëzel
 -/
-import measure_theory.function.simple_func_dense
-import measure_theory.function.l1_space
+import measure_theory.integral.set_to_l1
 import measure_theory.group.basic
 import analysis.normed_space.bounded_linear_maps
 import topology.sequences
```

Comment:
You should probably update drastically the file-level docstring.

Reply:
I updated it. It is not changed a lot. Since I kept all the intermediate steps of the integral construction, the docstring description still applies. The integral is still defined on simple functions etc.

I will experiment (for a possible future PR) with the possibility of removing all intermediate steps and defining directly the integral as `set_to_fun ...`.

Reply:
Yes, I was also wondering if all these steps were really necessary now that you have a general machinery in place. But for this PR I agree it is better to keep it like that.